addons/website_sale/views/templates.xml

```diff
@@ -2404,7 +2406,6 @@
                             </h3>
                         </t>
                     </t>
-                    <!-- FIXME KCV this is shown even when creating a new address when logged in -->
```

Comment:
This one is coming from master probably in 18.0 also. Created ticket https://www.odoo.com/odoo/project/49/tasks/4522138